package.json

```diff
@@ -15,5 +15,8 @@
   "homepage": "https://tc39.github.io/ecma262/",
   "dependencies": {
     "ecmarkup": "^3.0.0"
+  },
+  "devDependencies": {
+    "@alrra/travis-scripts": "^1.2.2"
```

Comment:
locking down dep versions doesn't give you any protection without shrinkwrap, and fails to communicate the intent to follow semver for updates when updating - it'd be best to use a `^` range, and then if you're concerned about updates, `npm shrinkwrap --dev` on top of that.